packages/lab/src/Table/stories/TableColumnRenderers.stories.mdx

```diff
@@ -0,0 +1,202 @@
+
+you can use the provided renderers by importing them directly from the lab like this: 
+
+<Usage>{'import { hvTextColumn } from "@hitachivantara/uikit-react-lab";'}</Usage>
+
+The UI Kit encourages the using of these renderers because: 
+
+- Ensures data fallbacks in case of missing values.
+- Includes an overflow tooltip in case of long values.
+- Handles embedded component cases.
+
+The following renderers are available:
+
+- `hvTextColumn`: Renders a string and include a overflow tooltip.
+- `hvNumberColumn`: Renders a number aligning it correctly and include a overflow tooltip.
+- `hvDateColumn`: Receives a date and formats it.
+- `hvExpandColumn`: Creates a column that include an expand button.
+- `hvTagColumn`: Creates a column rendering a tag component inside the cell.
+- `hvSwitchColumn`: A column that include a switch to toggle between two values.
+- `hvDropdownColumn`: A column that include a dropdown select between various values.
+- `hvProgressColumn`: Shows a bar in a column to visualize the compare between the actual value and max value.
+
+## All renderers table
+
+Complete sample table that uses most of the provided renderers.
+
+<Preview withToolbar>
+  <Story inline story={AllColumnRenderers} />
+</Preview>
+
+## Text column renderer
+
+This functions render text in the table.
+The column generated will be *left* aligned, the text truncated with a tooltip if too long or if missing it will be replaced with a long dash.
+
+parameters:
+- `col: HvTableColumnConfig`: The column configuration to modify.
+
+output:
+- `col: HvTableColumnConfig`: The modified column configuration with the parameters required to render the text.
+
+<Preview withToolbar>
+  <Story inline story={TextColumnRenderer} />
+</Preview>
+
+## Number column renderer
+
+This functions render numeric values in the table.
+The column generated will be *right* aligned and in case the value is missing it will be substituted by a 0.
+
+parameters:
+- `col: HvTableColumnConfig`: The column configuration to modify.
+
+output:
+- `col: HvTableColumnConfig`: The modified column configuration with the parameters required to render the text.
+
+<Preview withToolbar>
+  <Story inline story={NumberColumnRenderer} />
+</Preview>
+
+## Date column renderer
+
+This functions renders format string values as a date.
+The column generated will be *right* aligned and in case the value is missing it will be substituted by a 0.
+
+parameters:
+- `col: HvTableColumnConfig`: The column configuration to modify.
+- `format: String`: the format used to format the value the default value is `ISO8601`.
+
+output:
+- `col: HvTableColumnConfig`: The modified column configuration with the parameters required to render the text.
+
+<Preview withToolbar>
+  <Story inline story={DateColumnRenderer} />
+</Preview>
+
+## Expand column renderer
+
+This functions in conjunction with the `useHvRowExpand` hook renders a column with a button to open additional content in the table.
```

Comment:
We need to explain that this is to be used when we need to customise the default expand button created by `useHvRowExpand` (that is always in the first column, only supports plain text, etc.) and that it implies the use of `disableCreateExpandButton: true`.